export/securedrop_export/disk/cli.py

```diff
@@ -248,18 +295,147 @@ def unlock_luks_volume(self, volume: Volume, decryption_key: str) -> Volume:
 
         except subprocess.CalledProcessError as ex:
             raise ExportException(sdstatus=Status.DEVICE_ERROR) from ex
 
+    def _get_dev_mapper_entries(self) -> List[str]:
+        """
+        Helper function to return a list of entries in /dev/mapper/
+        (excluding `system` and `dmroot`).
+        """
+        try:
+            ls = subprocess.check_output(["ls", "/dev/mapper/"], stderr=subprocess.PIPE)
+            entries = ls.decode("utf-8").rstrip().split("\n")
+
+            return [r for r in entries if r not in _DEVMAPPER_SYSTEM]
+
+        except (subprocess.CalledProcessError, ValueError) as ex:
+            logger.error(f"Error checking entries in /dev/mapper: {ex}")
+            raise ExportException(sdstatus=Status.DEVICE_ERROR) from ex
+
+    def _attempt_get_unlocked_veracrypt_volume(self, device_name: str) -> MountedVolume:
+        """
+        Looks for an already-unlocked volume in /dev/mapper to see if the name matches
+        given device name.
+        Returns MountedVolume object if a drive is found. Otherwise, raises ExportException.
+        """
+        try:
+            devmapper_entries = self._get_dev_mapper_entries()
+            for item in devmapper_entries:
+                # Check it out with cryptsetup, see if it's a VeraCrypt/TrueCrypt drive.
+                # Example format (some lines ommitted for brevity):
+                #
+                # b'/dev/mapper/vc is active and is in use.\n  type:    TCRYPT\n  cipher:
+                # aes-xts-plain64\n keysize: 512 bits\n  key location: dm-crypt\n  device:
+                # /dev/sdc\n  sector size:  512\noffset:  256 sectors\n  size:
+                # 1968640 sectors\n  skipped: 256 sectors\n  mode:    read/write\n'
+                #
+                # (A mapped entry can also have a null device, if it wasn't properly removed
+                # from /dev/mapper using `cryptsetup close`.)
+                status = (
+                    subprocess.check_output(
+                        ["sudo", "cryptsetup", "status", f"/dev/mapper/{item}"]
+                    )
+                    .decode("utf-8")
+                    .split("\n  ")
+                )
+
+                logger.debug(f"{status}")
+
+                if "type:    TCRYPT" in status and f"device:  {device_name}" in status:
+                    logger.info("Unlocked VeraCrypt volume detected")
+                    volume = Volume(
+                        device_name=device_name,
+                        mapped_name=item,
+                        encryption=EncryptionScheme.VERACRYPT,
+                    )
+
+                    # Is it mounted?
+                    mountpoint = (
+                        subprocess.check_output(
+                            [
+                                "lsblk",
+                                f"/dev/mapper/{item}",
+                                "--noheadings",
+                                "-o",
```

Comment:
I think it's okay that we're `lsblk`ing directly here, since we don't use it so much that it's worth introducing something like <https://github.com/genalt/blkinfo>.  But why not use the output of `lsblk --json`?

Whatever the output format, I suggest long arguments for readability:

```suggestion
                                "--output",
```

Reply:
Ah!  I see that you've started this in rocodes/securedrop-export#1, which I support wholeheartedly.